config/packages/csrf.yaml

```diff
@@ -0,0 +1,11 @@
+# Enable stateless CSRF protection for forms and logins/logouts
+framework:
+    form:
+        csrf_protection:
+            token_id: submit
+
+    csrf_protection:
```

Comment:
this looks like a separate change

Reply:
Yes, this was just to get it working. I'm not really understanding the set-up of this project, because it has a symfony.lock but no composer.lock (although composer.lock is not gitignored). So I couldn't install phpunit/phpunit without also getting these recipe updates.